### ISSUE TITLE
断句问题

### ISSUE THREAD

**Issue by LeesimEverglow** [FunAudioLLM/CosyVoice#95]
用预训练音色，来使用文本训练语音，生成的音频断句节奏都有点怪，无法达到paper里演示的基本效果。

目前看音色是固定的开放几种，调整节奏只找到了随机推理种子来随机 torch 和cuda 随机seed的方式来调整断句节奏，但是测试效果不理想，太随机。

请问有调整断句和语音节奏的参数么？想更接近于paper效果级别

**Comment by aluminumbox** [comment]
no, this is frontend logic, you can cut sentence by yourself. all demo in paper is generated using the our released model

**Comment by LeesimEverglow** [comment]
> no, this is frontend logic, you can cut sentence by yourself. all demo in paper is generated using the our released model

您好 有推荐的效果比较好的推理种子数据么？ 即使是用和papaer里一样的断句，也无法达到一样的效果，是推理种子不一致而导致的么？

**Comment by LeesimEverglow** [comment]
最后使用了prompt的方式，学习一段语速节奏快的音频，之后说话的节奏听起来更正常了。

**Comment by limingchina** [comment]
> no, this is frontend logic, you can cut sentence by yourself. all demo in paper is generated using the our released model

Maybe such logic can be added to the webui.py so that to demo the project better?

**Comment by Yukiniro** [comment]
> no, this is frontend logic, you can cut sentence by yourself. all demo in paper is generated using the our released model

请问演示中的效果实现有考虑开源吗？
